_data/categories/meshes.yml

```diff
@@ -10,6 +10,19 @@
   prometheus: "No, Replaced by Harmony Controller"
   tracing: "Yes"
   encryption: "Yes"
+  
+  - name: App Mesh
+  desc: A multi-service mesh management plane to enable adoption, operation and development of service meshes and services running on them
```

Comment:
@ashiscs thanks for submitting this PR. The description used here is that of the Meshery project. Will you find and use a description for App Mesh instead?